easybuild/easyblocks/p/pytorch.py

```diff
@@ -360,6 +377,38 @@ def configure_step(self):
 
         pytorch_version = LooseVersion(self.version)
 
+        self.has_xml_test_reports = False
+        if pytorch_version >= '1.10.0':
+            out, ec = run_shell_cmd(self.python_cmd + " -c 'import xmlrunner'")
```

Comment:
This wasn't correctly ported to `run_shell_cmd`, fixing that in https://github.com/easybuilders/easybuild-easyblocks/pull/3664